CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 ------
 
 **ENHANCEMENTS**
+- Add support for Data Repository Associations with Pcluster Managed FSx for Lustre.
```

Comment:
- Add support for Data Repository Associations when using `PERSISTENT_2` as `DeploymentType` for a managed FSx for Lustre.